### DISCUSSION TITLE
Can no longer Scrobble

### DISCUSSION THREAD

**Discussion by Jules382** [rern/rAudio#1231] [Fixed Issues]
When I try to log in using my correct LastFM credentials I am getting an error "invalid method signature supplied". Does anyone know how to fix this please? I have 2 r-audios both using the latest version and both have the same error.

Many thanks, here's hoping.

Jules

**Comment by Vyacheslav-S** [comment]
I also tried to connect to my Last FM account and got an error:
``invalid method signature supplied``
I installed [mpdscribble](https://aur.archlinux.org/packages/mpdscribble) and it works.
I also installed the ListenBrainz sending client for MPD and it works fine https://github.com/rern/rAudio-1/discussions/1228.
